src/components/DacaRenewalQuestionManager/DacaRenewalQuestionManager.tsx

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable react/style-prop-object */
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, ScrollView, View } from 'react-native';
+import { Appbar } from 'react-native-paper';
+import { getAllQuestionsOfType, setCase, setClient } from 'database/queries';
+import { TextSubtitle, TextRegularWhite } from 'assets/fonts/Fonts';
+import { ButtonDarkBlue } from 'assets/Components';
+import {
+  Question,
+  Client,
+  QuestionManagerProps,
+  Case,
+  CaseType,
+  CaseStatus,
+} from 'types/types';
+import LargeInput from 'components/LargeInput/largeInput';
+import SmallInput from 'components/SmallInput/smallInput';
+import Dropdown from 'components/Dropdown/dropdown';
+import Calendar from 'components/Calendar/calendar';
+import Radio from 'components/Radio/radio';
+import { getCurrentClient } from 'database/auth';
+import { firestoreAutoId } from 'database/helpers';
+import { ButtonHeader, ButtonView } from './styles';
+
+const styles = StyleSheet.create({
+  container: {
+    width: '100%',
+    height: '100%',
+    backgroundColor: '#fff',
+    display: 'flex',
+    paddingTop: 10,
+  },
+});
+
+export default function DacaRenewalQuestionManager(
+  props: QuestionManagerProps,
+) {
+  const [allQuestions, setAllQuestions] = useState([] as Question[]);
+  const [currentAnswers, setCurrentAnswers] = useState(new Map());
```

Comment:
does reseting state to new Map() mean you're always discarding the constructed list of answers every time? Is the intended functionality to save answers here?

Reply:
fixed issue, doesn't have to do with this, the empty map is only on re-render